cmake/Modules/FeatureOptionsSetup.cmake

```diff
@@ -0,0 +1,14 @@
+# Helper functions for coupling add_feature_info() or CMAKE_DEPENDENT_OPTION() and option()
+
+function(feature_option _name _description _default)
+    string(CONCAT _desc "${_description}" "; default: " "${_default}")
```

Comment:
Such strings (not CMake code but output strings) break my eyes. I would reformat it:
```cmake
string(CONCAT _desc "${_description}" " (default: " "${_default})")
```

Reply:
@glassez fixed.